Kitodo/src/main/webapp/pages/inc_Prozessverwaltung/Prozesse_Liste.xhtml

```diff
@@ -36,9 +36,11 @@
     </h4>
 
     <!-- new process -->
-    <h:link outcome="/pages/ProzessverwaltungBearbeiten"
-            value="#{msgs.einenNeuenProzessAnlegen}"
-            rendered="#{(LoginForm.maximaleBerechtigung == 1 || LoginForm.maximaleBerechtigung == 2) and ProzessverwaltungForm.modusAnzeige!='vorlagen'}"/>
+    <h:commandLink action="#{ProzessverwaltungForm.newProcess}" immediate="true"
```

Comment:
PR https://github.com/kitodo/kitodo-production/pull/1142 removes this link, as it was decided that this method of task insert is incorrect. So this part can be ignored.

Reply:
I would go with @Beacze comment: do not change it now as this command link will be removed in the next pull request (or Beatrycze must adjust their pull request after this one is merged).